src/SourceBuild/Arcade/eng/common/templates/steps/source-build-run-tarball-build.yml

```diff
@@ -23,6 +23,7 @@ steps:
       --online
   workingDirectory: ${{ parameters.sourceFolder }}
   displayName: Build Tarball
+  timeoutInMinutes: 180
```

Comment:
I don't see this as necessary/valuable given the job's timeout is the same.  Consider removing.

Reply:
Removed.